soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -145,15 +156,15 @@ bool intel_adsp_ipc_send_message(const struct device *dev,
 	return true;
```

Comment:
mixing types ... shouldn't it be 0 ?

Reply:
yes, this was a cause of tests failing thanks